environments/sandbox.yml

```diff
@@ -172,6 +172,10 @@ cname:
     ttl: 300
     record: "sdshmcts-sbox.azurefd.net"
     shutter: false
+  - name: "toffee2"
+    ttl: 300
+    record: "test-sbox.azurefd.net"
```

Comment:
should have hmcts in the name as these are globally namespaced someone else could have this

Reply:
updated it with hmcts-test-sbox.azurefd.net